server/migrations/20220427204420-changeAtVersionColumn.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+    up: async queryInterface => {
+        queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.sequelize.query(
+                'ALTER TABLE public."AtVersion" RENAME COLUMN "atVersion" TO "name";',
+                { transaction }
+            );
+        });
+    },
+
+    down: async queryInterface => {
+        queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.sequelize.query(
+                'ALTER TABLE public."AtVersion" RENAME COLUMN "name" TO "atVersion";',
+                { transaction }
+            );
+        });
+    }
```

Comment:
It should be possible to combine all four migrations into one, that'll just make the migration more compact and easier to take in all at once.

Reply:
Does the migration work? Does the ID field auto-populate? If so, that's one of the biggest hurdles overcome, which would be sweet.

Reply:
I think we should also make a symmetrical set of changes to the Browser table, i.e. giving it an ID field and possibly a releasedAt field too. @s3ththompson are we going to track the dates where browser versions were released as well?

Reply:
> Does the migration work? Does the ID field auto-populate? If so, that's one of the biggest hurdles overcome, which would be sweet.

Yes, the IDs are auto-populated.

Reply:
> It should be possible to combine all four migrations into one, that'll just make the migration more compact and easier to take in all at once.

I was running into some issues with the ordering of changes, which is why I broke them out. I know multiple files isn't ideal, but they are easier to read, if anything.